Sources/swift-doctest/main.swift

```diff
@@ -81,6 +90,22 @@ struct SwiftDocTest: ParsableCommand {
 
         let consolidatedReport = Report.consolidation(of: reports)
         standardOutput.write(consolidatedReport.description.data(using: .utf8)!)
+        let failureList = consolidatedReport.results.filter({ result -> Bool in
+            do {
+                return try !result.get().ok
+            } catch {
+
+                return true
+            }
+        })
+        if (failureList.count > 0) {
```

Comment:
```suggestion
      if consolidatedReport.results.contains(where: { (try? $0.get().ok) != true }) {
```

Reply:
See review comment below. If we don't need the result of `map`, we should use `contains(where:)` instead.